package.json

```diff
@@ -116,6 +116,7 @@
     "postcss-cssnext": "^2.7.0",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0",
+    "sweetalert": "^1.1.3",
```

Comment:
K, for some reason I thought it worked automagically, haven't used shrinkwrap too extensively.